release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1780,7 +1761,7 @@ With this update, the CMO now properly propagates the external labels that you c
 
 * Previously, pod failures were artificially extending the validity period of certificates causing them to incorrectly rotate. With this update, the certificate validity period is correctly determined and the certificates are correctly rotated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2020484[*BZ#2020484*])
 
-* In {product-title} {product-version} the default cluster-wide pod security admission policy is set to `baseline` for all namespaces and the default warning level is set to `restricted`. Before this update, Operator Lifecycle Manager displayed pod security admission warnings in the `operator-marketplace` namespace. With this fix, reducing the warning level to `baseline` resolves the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088541[*BZ#2088541*])
+* In {product-title} {product-version} the default cluster-wide pod security admission policy is set to `privileged` for all namespaces and the default warning level is set to `restricted`. Before this update, Operator Lifecycle Manager displayed pod security admission warnings in the `operator-marketplace` namespace. With this fix, reducing the warning level to `baseline` resolves the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088541[*BZ#2088541*])
```

Comment:
```suggestion
* In {product-title} {product-version} the default cluster-wide pod security admission policy is set to `privileged` for all namespaces which are not part of the OpenShift system and the default warning level is set to `restricted`. Before this update, Operator Lifecycle Manager displayed pod security admission warnings in the `operator-marketplace` namespace. With this fix, reducing the warning level to `baseline` resolves the issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088541[*BZ#2088541*])
```

@michaelryanpeter  all namespaces prefixed with openshift and + a few of them are restricted but I do not think we need to describe them here. More info: 
[OCP 4.11 Documentation](https://github.com/openshift/openshift-docs/pull/48409)
[OCP 4.11 Release Notes](https://github.com/openshift/openshift-docs/pull/48402/files)

Reply:
This is just a summary and rewording of the doc text in the BZ. This is not meant to do anything other than provide a more customer friendly summary of the content of the doc text in the BZ.

Reply:
@michaelryanpeter the info is not accurate and I think we should link here the release notes that explain what was changed in the OCP 4.11 and why the warning is raised now.